iis-schemas/src/main/avro/eu/dnetlib/iis/referenceextraction/project/DocumentHash.avdl

```diff
@@ -0,0 +1,8 @@
+@namespace("eu.dnetlib.iis.referenceextraction.project.schemas")
+protocol IIS{
+
+    record DocumentHash {
+//      document hash value calculated as murmur3 hash of document title, abstract and text
```

Comment:
It was not particularly well thought decision. I've relied on Spark hashing functions and `functions.hash`, using murmur3, accepts an array of columns, so I though that it would be the simplest solution, not introducing any compound columns used only for hashing. But given your comment I've done tests of Spark hashing functions, the results are [here](https://github.com/openaire/iis/issues/1099). Using a compound column for hashing created as a concatenation of document's title, abstract and text, with `|` as separator, it turns out that `functions.sha1` was the best in terms of execution time. So I'll go with it as hashing function, this will also change the type of hash value to string.

Reply:
Thanks for this thorough comparison. It begs to become at least a blog post instead of a comment which may not reach the audience it desires :)